SPECS/reaper/reaper_build_caches.sh

```diff
@@ -0,0 +1,168 @@
+#! /bin/bash
+
+set -euxo pipefail
+
+# NOTE: EXECUTING with id=0 (root) CAUSES REAPER BUILD TO FAIL.
+if [[ $(id -u) -eq 0 ]]; then
+    echo "Must run script as non-root user!" >&2
+    exit 1
+fi
+
+# Cassandra reaper version for which to generate build caches.
+# (a) Modify VERSION="x.x.x" in script.
+# (b) Pass arguments to command line: ./reaper_build_caches.sh 3.1.1
+VERSION="3.1.1"
+if [ $? -gt 1 ]; then
```

Comment:
`$?` is the exit value of the previous command. I'd use either `$#` or simply the value of `$1`.

```suggestion
if [ -n $1 ]; then
```

Reply:
typo error, intention was $#.